packages/suite/src/components/wallet/Menu/components/useScrollRef.ts

```diff
@@ -0,0 +1,56 @@
+import { useCallback, useState, useEffect } from 'react';
+
+interface Dimensions {
+    height: number;
+    scrollHeight: number;
+    scrollTop: number;
+    hasScroll: boolean;
+}
+// top: 'x' in rect ? rect.x : rect.top,
+// left: 'y' in rect ? rect.y : rect.left,
+// x: 'x' in rect ? rect.x : rect.left,
+// y: 'y' in rect ? rect.y : rect.top,
+const getDimensions = (node: HTMLElement) => {
+    const rect = node.getBoundingClientRect();
+    return {
+        height: rect.height,
+        scrollHeight: node.scrollHeight,
+        scrollTop: node.scrollTop,
+        hasScroll: Math.round(rect.height) < node.scrollHeight,
+    };
+};
+
+export const useScrollRef = () => {
```

Comment:
move this file to hooks?

Reply:
this is specific hook used only here in menu, i was thinking about moving it but imho it doesnt make sense to separate this code since it's not reusable

Reply:
done